### PR TITLE
Fix Symfony 4.4 compatibility

### PR DESCRIPTION
**Pull request by VincentLanglet**
## Subject

This config doesn't work for Sf4.4 and we don't have conflict so far.

## Changelog

```markdown
### Fixed
- Symfony 4.4 compatibility
```

### CODE REVIEW

**Comment by VincentLanglet** [comment]
As a proof, https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1680 build of Symfony 4.4 is working

**Comment by VincentLanglet** [comment]
We'll need to do a PR like https://github.com/sonata-project/exporter/pull/605 with a conflict later.

File: src/Bridge/Symfony/Resources/config/doctrine_orm.php [hunk 1]
```diff
@@ -16,10 +16,11 @@
 use Sonata\Doctrine\Adapter\ORM\DoctrineORMAdapter;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
+    // Use "service" function for creating references to services when dropping support for Symfony 4.4 (and add conflict)
     $containerConfigurator->services()
 
         ->set('sonata.doctrine.adapter.doctrine_orm', DoctrineORMAdapter::class)
             ->args([
-                service('doctrine'),
+                new ReferenceConfigurator('doctrine'),
```
Comment (dmaicher) [inline src/Bridge/Symfony/Resources/config/doctrine_orm.php:24]: isn't there a `use` statement missing for this class?

Comment (VincentLanglet) [reply src/Bridge/Symfony/Resources/config/doctrine_orm.php:24]: We're in the same namespace

Comment (dmaicher) [reply src/Bridge/Symfony/Resources/config/doctrine_orm.php:24]: ah indeed :smile: missed that

**Review by dmaicher — APPROVED** [review]

**Comment by jordisala1991** [comment]
Why dont we do the conflict thing here too?

**Comment by VincentLanglet** [comment]
> Why dont we do the conflict thing here too?

Because we released a not working 2.0.0 version which can be installed with Sf 4.4
So I need to release a fix 2.0.1
And then we can do the conflict thing.

There is no way to prevent user to install 2.0.0 since it's released :(

**Comment by jordisala1991** [comment]
We can remove the tag, or is it not possible?

**Comment by VincentLanglet** [comment]
> We can remove the tag, or is it not possible?

I'm note sure it will be enough.
I'm not an expert at all ; I just heard it's really hard to completely delete a release.
But we can ask for help about it if you prefer this way.
It seems simpler this way

**Review by jordisala1991 — APPROVED** [review]
If it is simpler this way, lets do it like this. Later we can depend on 2.1 right?

**Pull request merged**